src/components/common/SafeTokenWidget/index.tsx

```diff
@@ -28,25 +31,22 @@ export const getSafeLockingAddress = (chainId: string): string | undefined => {
   return SAFE_LOCKING_ADDRESS[chainId]
 }
 
-const canRedeemSep5Airdrop = (allocation?: Vesting[]): boolean => {
-  const sep5Allocation = allocation?.find(({ tag }) => tag === 'user_v2')
-
-  if (!sep5Allocation) {
-    return false
-  }
-
-  return !sep5Allocation.isRedeemed && !sep5Allocation.isExpired
-}
-
```

Comment:
Nice!

Reply:
Let's use the same rules for rounding in the header as we use in the safe{Pass} to have the exact numbers